src/config/lang/english.go

```diff
@@ -214,6 +214,7 @@ zarf init --git-push-password={PASSWORD} --git-push-username={USERNAME} --git-ur
 	CmdPackageCreateFlagMaxPackageSize     = "Specify the maximum size of the package in megabytes, packages larger than this will be split into multiple parts. Use 0 to disable splitting."
 	CmdPackageCreateFlagSigningKey         = "Path to private key file for signing packages"
 	CmdPackageCreateFlagSigningKeyPassword = "Password to the private key file used for signing packages"
+	CmdPackageCreateFlagDifferential       = "Build a differential package that only contains the changes local resources and new remote resources since the last build"
```

Comment:
```suggestion
	CmdPackageCreateFlagDifferential       = "Build a package that only contains the differential changes from local resources and new remote resources since the last build"
```